apps/docs/src/app/core/component-docs/splitter/examples/api/splitter-api-example.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, ViewChild } from '@angular/core';
+
+import { SplitterComponent, SplitterSplitPaneComponent } from '@fundamental-ngx/core/splitter';
+
+
+@Component({
+    selector: 'fd-slider-api-example',
+    templateUrl: './splitter-api-example.component.html'
+})
+export class SplitterApiExampleComponent {
+    @ViewChild(SplitterComponent)
+    splitter: SplitterComponent;
+
+    @ViewChild('paneOne')
+    paneOne: SplitterSplitPaneComponent
+
+    paneOneId = 'one';
```

Comment:
Can we remove `paneOneId` and `paneTwoId` variables, and remove the data binding in the template?